sklearn/cluster/k_means_.py

```diff
@@ -724,7 +734,7 @@ class KMeans(BaseEstimator, ClusterMixin, TransformerMixin):
 
     def __init__(self, n_clusters=8, init='k-means++', n_init=10, max_iter=300,
                  tol=1e-4, precompute_distances='auto',
-                 verbose=0, random_state=None, copy_x=True, n_jobs=1):
+                 verbose=False, random_state=None, copy_x=True, n_jobs=1):
```

Comment:
As I said, I would not change that. I prefer integer verbosity and in particular I'm not sure if the tests cover all verbosity settings.